games-util/springlobby/springlobby-0.269.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+WX_GTK_VER="3.0"
+inherit cmake-utils gnome2-utils wxwidgets
```

Comment:
Looks like *gnome2-utils* is only used to update icon cache, it shouldn't even work with EAPI-7. You should use *xdg-utils* and adjust the `icon_cache_update` calls below.